web_src/js/components/RepoActionView.vue

```diff
@@ -405,6 +405,7 @@ export function initRepositoryActionView() {
 
 .job-group-section .job-brief-list .job-brief-item .job-brief-link span {
   margin-right: 8px;
+  padding-left: 5px;
```

Comment:
```suggestion
  margin: 0 8px;
```

Equal margin on both sides, and I do think 5px is too few.